src/components/campax/Initiative.js

```diff
@@ -205,7 +205,7 @@ export default function Register(props) {
   };
 
   useEffect(() => {
-    register({ name: "postcode" });
+    register("postcode");
```

Comment:
do we really need that code? the postcode is registered anyway, isn't it?

Reply:
Yes, it should be, I'll remove it. Any idea what widget/config to run to check?

Reply:
grep campax config/Initiative ;)

28 29 30